activate

```diff
@@ -71,25 +82,45 @@ serial="${serial%"${serial##*[![:space:]]}"}"
 # detect live system
 live="false"
 if grep -qw "endless.live" /proc/cmdline; then
-    image="true"
+    live="true"
+fi
+
+
+# We don't want dummy serials sent
+if [ "${serial}" == "To be filled by O.E.M." ]; then
```

Comment:
Meh? I doubt they can complain at us very convincingly if they didn't fill their BIOS in correctly... :) We're not going to call them back unless it's an eosoem image, so it should only be the ASUS systems they put EOS onto that do this.